tensorflow/tensorflow.bzl

```diff
@@ -2653,6 +2653,29 @@ def gpu_py_test(
 def cuda_py_test(*args, **kwargs):
     gpu_py_test(*args, **kwargs)
 
+def trt_py_test(test_names, tags):
```

Comment:
This file refers tf-trt as tensorrt, suggest s/trt_py_test/tensorrt_py_test/

Reply:
Done